appinventor/components/src/com/google/appinventor/components/runtime/util/WebRTCNativeMgr.java

```diff
@@ -283,6 +317,7 @@ public void initiate(ReplForm form, Context context, String code) {
     rtcConfig.continualGatheringPolicy = ContinualGatheringPolicy.GATHER_CONTINUALLY;
     peerConnection = factory.createPeerConnection(rtcConfig, new MediaConstraints(),
       observer);
+    request = new HttpGet("http://" + rendezvousServer2 + "/rendezvous2/" + rCode + "-s?http11=true");
```

Comment:
Is there a way we can indicate in the client that we want HTTP 1.0 or HTTP 1.1? This seems like a rather poor hack for something built into the spec.